### ISSUE TITLE
Adding Quick-Sort

### ISSUE THREAD

**Issue by Sanskar0820** [serverx-org/DSA-MASTERY#24]
I want to work on this issue through hacktoberfest2024

**Comment by Pariyul** [comment]
I want to work on this issue through hacktoberfest2024

**Comment by gautamankoji** [comment]
yeah you can work submit an pull request and it will be reviewed and merged
